katharsis-core/src/main/java/io/katharsis/resource/registry/repository/adapter/RelationshipRepositoryAdapter.java

```diff
@@ -0,0 +1,290 @@
+    }
+
+    @SuppressWarnings("rawtypes")
+	public JsonApiResponse removeRelations(T source, Iterable<J> targetIds, String fieldName, QueryAdapter queryAdapter) {
+    	RepositoryRequestFilterChainImpl chain = new RepositoryRequestFilterChainImpl() {
+
+			@Override
+			protected JsonApiResponse invoke(RepositoryFilterContext context) {
+				RepositoryRequestSpec request = context.getRequest();
+				Object source = request.getEntity();
+				Iterable<?> targetIds = request.getIds();
+				String fieldName = request.getRelationshipField();
+				QueryAdapter queryAdapter = request.getQueryAdapter();
+		    	if (isAnnotated) {
+		            ((AnnotatedRelationshipRepositoryAdapter) relationshipRepository)
+		                .removeRelations(source, targetIds, fieldName, queryAdapter);
+		        } else if(relationshipRepository instanceof QuerySpecRelationshipRepository){
+		        	((QuerySpecRelationshipRepository)relationshipRepository).removeRelations(source, targetIds, fieldName);
+		        } else {
+		            ((RelationshipRepository) relationshipRepository).removeRelations(source, targetIds, fieldName);
+		        }
+		        return new JsonApiResponse();
+			}
+		};
+		RepositoryRequestSpec requestSpec = RepositoryRequestSpecImpl.forRelation(moduleRegistry.getResourceRegistry(), HttpMethod.DELETE, source, queryAdapter, targetIds, fieldName, resourceInformation.getResourceClass());
+		return chain.doFilter(newRepositoryFilterContext(requestSpec));
+    }
+
+	@SuppressWarnings("rawtypes")
+	public JsonApiResponse findOneTarget(I sourceId, String fieldName, QueryAdapter queryAdapter) {
+    	RepositoryRequestFilterChainImpl chain = new RepositoryRequestFilterChainImpl() {
+
+			@Override
+			protected JsonApiResponse invoke(RepositoryFilterContext context) {
+				RepositoryRequestSpec request = context.getRequest();
+				Serializable sourceId = request.getId();
+				String fieldName = request.getRelationshipField();
+				QueryAdapter queryAdapter = request.getQueryAdapter();
+				
+		    	Object resource;
+		        if (isAnnotated) {
+		            resource = ((AnnotatedRelationshipRepositoryAdapter) relationshipRepository)
+		                .findOneTarget(sourceId, fieldName, queryAdapter);
+		        } else if(relationshipRepository instanceof QuerySpecRelationshipRepository){
+		        	QuerySpecRelationshipRepository querySpecRepository = (QuerySpecRelationshipRepository) relationshipRepository;
+		        	Class<?> targetResourceClass = querySpecRepository.getTargetResourceClass();
+		        	resource = querySpecRepository.findOneTarget(sourceId, fieldName, request.getQuerySpec(targetResourceClass));
+		        } else {
+		            resource = ((RelationshipRepository) relationshipRepository)
+		                .findOneTarget(sourceId, fieldName, request.getQueryParams());
+		        }
+		        return getResponse(relationshipRepository, resource, request);
+			};
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)